Dockerfile

```diff
@@ -1,13 +1,32 @@
-FROM python:3.9
+FROM python:3.9-slim
 
 WORKDIR /code
 
+RUN \
+	apt update \
+	&& apt install -y \
+	# GDAL headers are required for fiona, which is required for geopandas.
+	# Also gcc is used to compile C++ code.
+	libgdal-dev g++ \
+	# libspatialindex is required for rtree.
+	libspatialindex-dev \
+	# Remove package index obtained by `apt update`.
+	&& rm -rf /var/lib/apt/lists/*
+
 COPY ./requirements.txt /code/requirements.txt
 
-RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+RUN \
+	pip install --no-cache-dir --upgrade -r /code/requirements.txt \
+	# For some reason, --no-cache-dir doesn't work. :(
+	&& rm -rf ~/.cache/pip
+
+ENV RIDE2GO_TOKEN=''
+
+EXPOSE 8000
 
 COPY ./app /code/app
+COPY ./static /code/static
 COPY config /code
 COPY logging.conf /code
 
-CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80"]
\ No newline at end of file
+CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
Have a look at the latest Dockerfile. I renamed the 3 Dockerfiles. The main Dockerfile is using gunicorn.

Reply:
I'm confused. On `main`, `Dockerfile` doesn't have a `CMD` anymore. What are `Dockerfile_gunicorn` & `Dockerfile_uvicorn` for?